packages/transaction-controller/src/types.ts

```diff
@@ -164,6 +188,24 @@ type TransactionMetaBase = {
    */
   submittedTime?: number;
 
+  /**
+   * [swaps-only]
+   * The symbol of the token being swapped.
+   */
+  sourceTokenSymbol?: string;
+
+  /**
+   * [swaps-only]
+   * The metadata of the swap transaction.
+   */
+  swapMetaData?: Record<string, unknown>;
+
+  /**
+   * [swaps-only]
+   * The value of the token being swapped.
```

Comment:
There is no info about these [swap-only] properties in extension, so descriptions are totally my guess.